kubernetes/process-manifest.rb

```diff
@@ -0,0 +1,94 @@
+require "yaml"
+require "fileutils"
+
+if ARGV.count != 3
+  puts "usage: #{$0} container-manifests-dir velum-source-code-dir salt-states-dir"
+  Process.exit 1
+end
+
+CONTAINERS_MANIFESTS_DIR = ARGV[0]
+VELUM_SOURCE_CODE_DIR = ARGV[1]
+SALT_STATES_DIR = ARGV[2]
+
+def container_volume(name:, path:)
+  {
+    "name" => volume_name(name: name),
+    "mountPath" => path
+  }
+end
+
+def patch_containers(yaml)
+  yaml["spec"]["volumes"] ||= []
+  yaml["spec"]["containers"].each do |container|
+    if container["image"] =~ /^sles12\/velum/
+      container["image"] = "sles12/velum:development"
+    elsif container["image"] =~ /^sles12/
+      container["image"] = "docker-testing-registry.suse.de/#{container["image"]}"
+    end
+    container["env"] ||= []
+    container["env"].each do |envvar|
+      envvar["value"] = "development" if envvar["name"] == "RAILS_ENV"
```

Comment:
could we also keep the port 3000 here? port 80 is fine as well, but i wonder why we don't stick with 3000